openapi2conv/openapi2_conv.go

```diff
@@ -242,6 +242,9 @@ func ToV3SchemaRef(schema *openapi3.SchemaRef) *openapi3.SchemaRef {
 	for k, v := range schema.Value.Properties {
 		schema.Value.Properties[k] = ToV3SchemaRef(v)
 	}
+	if schema.Value.AdditionalProperties != nil {
```

Comment:
`AdditionalProperties` can also be a boolean.

Reply:
What do you mean? It's defined as `AdditionalProperties *SchemaRef`
Do you mean AdditionalPropertiesAllowed??

Reply:
What change are you requesting?

Reply:
Oh you're right I forgot that this is handled with multijson tags.